trailblazer-activity-dsl-linear.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   end
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "trailblazer-activity", ">= 0.12.0", "< 1.0.0"
+  spec.add_dependency "trailblazer-activity", ">= 0.12.2", "< 0.13.0"
```

Comment:
The second "upper" requirement can be a PITA - any reasons not to leave it at `< 1.0.0`?

Reply:
Ah ok. I thought about minor version fixes, no other reasons. Reverted it.